### ISSUE TITLE
[Feature Request] Mouseless workflow for popup

### ISSUE THREAD

**Issue by ris58h** [mdnsk/leo-translate#18]
It would be great to work with leo-translate's popup completely mouseless.
Workflow:
1. Open extension's popup with shortcut (CTRL+SHIFT+L for example). We could use [commands API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/commands) for this. Note: at this moment in Firefox it's not possible for user to rebind commands, so collision with another extension's command is possible.
2. Enter the text to translate then get the translation.
3. Play the translation. It's possible to focus _play_ button by tab (we need to add some css style for focused buttons because at this time focused and unfocused buttons look the same). Autoplay could work too.
4. Press _Esc_ to close the popup (There is a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1435687) in Firefox with focusing document back but I hope they will fix it soon).

Bottom line:
- shortcut for the popup
- style for focused button

What do you think about it?
